code/lib/cli/src/automigrate/fixes/addons-api.test.ts

```diff
@@ -0,0 +1,44 @@
+import { addonsAPI } from './addons-api';
+import type { StorybookConfig } from '@storybook/types';
+import type { JsPackageManager } from '@storybook/core-common';
+import { expect, describe, it } from 'vitest';
+
+const checkAddonsAPI = async ({
+  packageManager,
+  mainConfig = {},
+  storybookVersion = '7.0.0',
+}: {
+  packageManager?: Partial<JsPackageManager>;
+  mainConfig?: Partial<StorybookConfig>;
+  storybookVersion?: string;
+}) => {
+  return addonsAPI.check({
+    packageManager: packageManager as any,
+    storybookVersion,
+    mainConfig: mainConfig as any,
+  });
+};
+
+describe('check function', () => {
+  it('should return { usesAddonsAPI: true } if @storybook/addons is installed', async () => {
+    await expect(
```

Comment:
await is not neccesary here right, since you are using resolves?

Reply:
if you use `expect(...).resolves.` you have to await it: https://vitest.dev/api/expect#resolves